src/manifest/dbtProject.ts

```diff
@@ -1005,18 +1006,20 @@ select * from renamed
     }
     const columns: Record<string, ColumnMetaData> = {};
     Object.entries(node.columns).forEach(([k, v]) => {
-      columns[k.toLowerCase()] = v;
+      columns[getColumnNameByCase(k)] = v;
```

Comment:
@AdiGajbhiye @mdesmet @suryaiyer95 here we are merging columns from local manifest and from db. This particular line is creating map of column name and column from manifest nodes. Since this actually takes column names from schema.yml, i am thinking we will not need this conversion `getColumnNameByCase` here. If user has lower case column name in schema.yml and from db if it is upper case, and if i dont do this conversion, both columns will not be matched and we might see duplicate columns. What do you guys think if I leave this change as it is? Will this have any side effects?

Reply:
So duplicates will be created right in this case?

Reply:
We should Ideally match making them more lower case treating existing name in schema.yml as priority? @mdesmet what do you think?

Reply:
That's what we discussed indeed. If is not a quoted identifier, the matching should occur in lowercase and we should prefer the yaml identifier if it is already there.

Reply:
ok, this is already handled in that way. I have modified the variable names to be more clear.